plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -1468,6 +1468,10 @@ public void testFilters()
                 .put("ipv6_column", "2001:db8:0:0:1:0:0:1")
                 .buildOrThrow());
 
+        assertQuery("SELECT count(*) FROM filter_pushdown WHERE keyword_column <> ''", "VALUES 1");
+        assertQuery("SELECT count(*) FROM filter_pushdown WHERE keyword_column = ''", "VALUES 0");
+        assertQuery("SELECT count(*) FROM filter_pushdown WHERE keyword_column = '0'", "VALUES 0");
+        assertQuery("SELECT count(*) FROM filter_pushdown WHERE text_column != ''", "VALUES 1");
```

Comment:
Could you add more tests for other types?

Reply:
Thanks for your time, added more test cases for other types.